### ISSUE TITLE
Dephy always logging 

### ISSUE THREAD

**Issue by Katharine-Walters** [neurobionics/opensourceleg#243]
## 🐛 Bug Report

FlexSEA DebugLog is always being created regardless of the value of the dephy_log optional argument in the add_joint function. 

## 🔬 How To Reproduce

Setting dephy_log = True and dephy_log = False results in the same behavior

### Code sample

self.dev = OpenSourceLeg(frequency=1000, file_name="MBLUEankle_left_actuator_log.log")
self.dev.add_joint(name="ankle", gear_ratio=9, port=self.actuator_port, dephy_log=False)

### Environment

RPi 5
Python 3.11.2

### Screenshots

<img width="372" alt="Screenshot 2024-06-21 at 12 18 17 PM" src="https://github.com/neurobionics/opensourceleg/assets/111811694/275e6588-2e27-451e-b512-b911efba1f9b">

## 📈 Expected behavior

dephy_log = False does not create DebugLog file

**Comment by Katharine-Walters** [comment]
Upon further investigation, I have discovered a few things: 
-The "dephy_log" bool is passed to the "debug" optional argument in the FlexSEA constructor. Is it unclear from the documentation what this controls, and I was unable to discern a change in behavior/logging when the bool was set to True or False.
-I was able to disable the dephy logging by setting the debug_level=6, which disables the logging in the FlexSEA constructor. 

I recommend implementing the following behavior: 
-Set default debug_level=4, which is the default logging level in the FlexSEA constructor. 
-When dephy_log = False, set debug_level=6, which will disable Dephy logging entirely.
